<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1708</width>
    <height>1090</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>821</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_prcs">
    <attribute name="title">
     <string>进程调度</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>基于优先级的时间片轮转</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>多级反馈队列轮转</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reset">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>选择算法：</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_timePiece">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>741</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>数据输入</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>到达时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>运行时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>时间片大小：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>序号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_arriveTime">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_runTime">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_timePiece">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_index">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_priority_1">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>优先级：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_priority_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>121</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_priority_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_priority_1">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>69</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_priority_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>120</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>2:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_priority_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>170</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>3:</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_11</zorder>
     <zorder>label_1</zorder>
     <zorder>lineEdit_arriveTime</zorder>
     <zorder>lineEdit_runTime</zorder>
     <zorder>lineEdit_timePiece</zorder>
     <zorder>lineEdit_index</zorder>
     <zorder>lineEdit_priority_1</zorder>
     <zorder>label_12</zorder>
     <zorder>pushButton_delete</zorder>
     <zorder>pushButton_add</zorder>
     <zorder>lineEdit_priority_2</zorder>
     <zorder>lineEdit_priority_3</zorder>
     <zorder>label_priority_1</zorder>
     <zorder>label_priority_2</zorder>
     <zorder>label_priority_3</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>作业调度</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>60</y>
     <width>771</width>
     <height>361</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>到达时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>运行时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>优先级</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>就绪队列：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
